Homework_2.sql

```diff
@@ -0,0 +1,33 @@
+USE baseball;
+SHOW TABLES;
+
+#calculation of annual average
+DROP TABLE IF EXISTS Annual_Average;
+CREATE TABLE Annual_Average as
+select b.batter, YEAR(g.local_date),
+    sum(Hit)/nullif(sum(atBat),0) as annual_avg
+    FROM game g JOIN batter_counts b on g.game_id = b.game_id group by b.batter, year(g.local_date)
+    order by  1,2;
+
+#calculation of historic average
+DROP TABLE IF EXISTS Historic_Average;
+CREATE TABLE Historic_Average as
+select b.batter,
+    sum(Hit)/nullif(sum(atBat),0) as Historic_avg
+    FROM  batter_counts b group by b.batter
+    order by  1,2;
```

Comment:
Major: The order by in this statement should not include the 2nd column. 
Minor: order by could specify the column names. 
Major: Please verify the results with a "select * from Historic_Average where batter=110029 limit 0,30;" statement

Reply:
updated column names and added "select * from Historic_Average where batter=110029 limit 0,30;" statement to test file.